app/Jobs/ProcessRulesJob.php

```
@@ -0,0 +1,155 @@
+     * The maximum number of unhandled exceptions to allow before failing.
+     */
+    public int $maxExceptions = 3;
+
+    /**
+     * The number of seconds the job can run before timing out.
+     */
+    public int $timeout = 300; // 5 minutes
+
+    private User $user;
+
+    private array $ruleIds;
```

PHPStan (2.1.15)

Property App\Jobs\ProcessRulesJob::$ruleIds type has no value type specified in iterable type array.
See: https://phpstan.org/blog/solving-phpstan-no-value-type-specified-in-iterable-type

(missingType.iterableValue)